### PR TITLE
NODE-5399 Node 5.0 Migration to wcli scripts

### PR DESCRIPTION
**Pull request by osix**

### CODE REVIEW

File: scripts/init [hunk 1]
```diff
@@ -1,46 +1,42 @@
...
-    args="$args -P $WALLARM_API_PORT"
+    args="$args -port $WALLARM_API_PORT"
   fi
 
   if [ x"$WALLARM_API_USE_SSL" = x"false" ] \
      || [ x"$WALLARM_API_USE_SSL" = x"False" ] \
      || [ x"$WALLARM_API_USE_SSL" = x"no" ]
   then
-    args="$args --no-ssl"
+    args="$args -ssl false"
```
Comment (qqshka) [inline scripts/init]:
```suggestion
    args="$args -ssl=false"
```

File: scripts/init [hunk 2]
```diff
@@ -1,46 +1,42 @@
...
   fi
 
   if [ x"$WALLARM_API_CA_VERIFY" = x"false" ] \
      || [ x"$WALLARM_API_CA_VERIFY" = x"False" ] \
      || [ x"$WALLARM_API_CA_VERIFY" = x"no" ]
   then
-    args="$args --no-verify"
+    args="$args -ssl-verify false"
```
Comment (qqshka) [inline scripts/init]:
```suggestion
    args="$args -ssl-verify=false"
```

File: scripts/init [hunk 3]
```diff
@@ -61,7 +57,7 @@ register_node() {
 }
 
 sync_node() {
```
Comment (qqshka) [inline scripts/init]: Just remove it and usage of it in the script.

File: scripts/init [hunk 4]
```diff
@@ -1,46 +1,42 @@
 #!/bin/bash
 
 register_node() {
-  args="--batch --no-sync"
+  args="init register"
```
Comment (qqshka) [inline scripts/init]: ` /opt/wallarm/register-node` is a shell script and it already contains `init` argument, so I think it should be like this (with this one you provide arguments for register job)
```suggestion
  args="job:register"
```

File: scripts/init [hunk 5]
```diff
@@ -165,7 +146,6 @@ printenv >> /etc/environment
 configure_services
 #taken from https://unix.stackexchange.com/a/71511
 { register_node |& tee /dev/fd/3 | grep -q 'Label "group" is required for this registration type' && exit; } 3>&1
```
Comment (qqshka) [inline scripts/init]: We grep here for particular message in the logs, `wcli` use different message for that, we have to check which exactly message we use.

Comment (nedvna) [inline scripts/init]:
```suggestion
{ register_node |& tee /dev/fd/3 | grep -q 'label "group" is required for this registration type' && exit; } 3>&1
```

Looks like not much has changed :)

File: scripts/init
Comment (nedvna) [inline scripts/init:1]: In line 52 here we're passing node UUID. I can't seem to find the proof that this is still supported.  
Should we remove this feature from init, or handle it somehow else?
@qqshka

Comment (qqshka) [reply scripts/init:1]: we don't support it anymore, we should remove it from init.

**Comment by nedvna** [comment]
@xDmitriev approval is needed

**Review by xDmitriev — APPROVED** [review]

**Comment by nedvna** [comment]
@xDmitriev thank you, merged  
Looks like I don't have permissions to switch the repo default branch to `stable/5.0` though :))

**Comment by nedvna** [comment]
@qqshka fyi we can now switch to `stable/5.0` in AIO repo

**Pull request merged**